src/Database/Pivot.php

```diff
@@ -1,17 +1,24 @@
 <?php namespace Winter\Storm\Database;
 
 use Illuminate\Database\Eloquent\Model as ModelBase;
-use Illuminate\Database\Eloquent\Builder as BuilderBase;
+use Winter\Storm\Support\Str;
 
 class Pivot extends Model
```

Comment:
Instead of replicating a the AsPivot trait in our code, can we just implement it in our Pivot model and then override any methods that need tweaking?

https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Eloquent/Relations/Concerns/AsPivot.php